### ISSUE TITLE
Reporting 404s to raygun

### ISSUE THREAD

**Issue by RobinClowers** [Cobenian/raygun#13]
Right now if your phoenix app 404s, the exception gets sent to raygun. I'm not 100% sure, but this seems wrong, since the exception is caught by `Phoenix.Endpoint.RenderErrors`. Would it be possible to catch errors higher up somehow? Or am I missing some configuration that will avoid this?

**Comment by brweber2** [comment]
It might not be a bad idea to make this a configuration option, especially if a lot of bots are hitting your site with garbage URLs. I can't think of a work around at the moment so I'll look into putting a proper solution in place.

Thanks!

**Comment by RobinClowers** [comment]
Yeah, I can imagine situations where you would want to see your 404s, but for us it's mostly just noise. Thanks for getting back to me.
